contact.html

```diff
@@ -4,101 +4,190 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Contact Form</title>
-    <link rel="stylesheet" href="contact.css">
+    <title>Contact page</title>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" />
+    <style>
```

Comment:
Maybe you could extract this into a separate css file `contact.css` and then import it into the html via a link like so:
```
<link rel="stylesheet" href="contact.css">
```